extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/namespace/URIExtractionNamespace.java

```diff
@@ -217,13 +217,12 @@ private DelegateParser(
           key,
           input
       ).toString(); // Just in case is long
-      final String val = Preconditions.checkNotNull(
-          inner.get(value),
-          "Value column [%s] missing data in line [%s]",
-          value,
-          input
-      ).toString();
-      return ImmutableMap.<String, String>of(k, val);
+      final Object val = inner.get(value);
+      if (val == null) {
+        // Skip nulls
```

Comment:
I don't think there should be a debug message here, the whole point is that it could be normal for this to happen. But I will add a log.info message at the end of loading that calls out the number of lines read and entries parsed, so if those are unexpectedly different you can do something about it.